connectors/sources/confluence.py

```diff
@@ -258,9 +279,18 @@ async def paginated_api_call(self, url_name, **url_kwargs):
                         self.host_url,
                         links.get("next")[1:],
                     )
+            except ServerDisconnectedError:
+                raise
+            except ClientResponseError:
+                raise
```

Comment:
```suggestion
            except (ServerDisconnectedError, ClientResponseError) as e:
                raise
```

If two exceptions are being handled the exact same way, no need to define the handling twice.